website/docs/foundations/typography/partials/guidelines/overview.md

```diff
@@ -0,0 +1,23 @@
+
+## Font stack
+
+Helios uses the default system fonts available on each OS. We opt for system fonts because they are stable and offer various styles that support internationalization, code, tabular data, data viz, etc. In addition, they help improve performance by reducing the number of resources to be loaded, which is important for large applications like those at HashiCorp.
+
+While the text might display differently across each OS, contextually to a user of that OS, it won’t seem out of place.
+
+
+### Sans-serif
+
+![sans-serif on mac](/assets/foundations/typography/sans-serif-mac.png =660x*)
+
+![sans-serif on windows](/assets/foundations/typography/sans-serif-windows.png =660x*)
+
+![sans-serif on linux](/assets/foundations/typography/sans-serif-linux.png =660x*)
```

Comment:
Seeing all of these in context together to left alignment of the text within the Linux description is really jarring. Everything else that is contained in these blocks is center aligned